### DISCUSSION TITLE
Is there a way to prevent the generator of the versions.properties file when using a version catalog?

### DISCUSSION THREAD

**Discussion by jadlr** [Splitties/refreshVersions#717] [Q&A] [answered]
Hi all, 

Is there a way to prevent the generation of the `versions.properties` file when using a version catalog? I'm using a version catalog in a toml file. When adding the refreshVersions plugin and running a gradle sync an empty `versions.properties` is always generated. Is there a way to turn that off?

Thanks!

**Comment by Gounlaf** [comment, marked as answer]
You might find some answers in this issue: https://github.com/Splitties/refreshVersions/issues/640

**Reply by jadlr** [reply]
Wonderful! Thank you :100:
